modules/react/pagination/stories/pagination.stories.mdx

```diff
@@ -212,172 +119,142 @@ provide a configuration as in the example below.
   rangeSize={3}
   onPageChange={pageNumber => console.log(pageNumber)}
 >
-  {/* child components go here */}
+  {/* Child components */}
 </Pagination>
 ```
 
-However, if you'd like to provide a model, you can follow the example below. This is called the
-hoisted model pattern. Note that in this pattern, the only props needed are `model` and
-`aria-label`.
+Alternatively, you may pass in a model using the hoisted model pattern.
 
 ```tsx
-const modelConfig = {
+const model = usePaginationModel({
   lastPage: 100,
   initialCurrentPage: 6,
   rangeSize: 3,
   onPageChange: pageNumber => console.log(pageNumber),
-};
-const model = usePaginationModel(modelConfig);
+});
 
 return (
   <Pagination aria-label="Pagination" model={model}>
-    {/* child components go here */}
+    {/* Child components */}
   </Pagination>
 );
 ```
 
-#### Component Props
+#### Props
 
-Given that there are two ways to configure `Pagination`, there are also two props tables. For the
-configuration pattern, follow this table:
+Undocumented props are spread to the underlying `<nav>` element.
 
-| name               | type                           | required?  | default | recommended                              |
-| ------------------ | ------------------------------ | ---------- | ------- | ---------------------------------------- |
-| aria-label         | `string`                       | ✅ `true`  | n/a     | "Pagination" (and translated equivalent) |
-| lastPage           | `number`                       | ✅ `true`  | n/a     | n/a                                      |
-| initialCurrentPage | `number`                       | 🚫 `false` | 1       | n/a                                      |
-| rangeSize          | `number`                       | 🚫 `false` | 5       | n/a                                      |
-| firstPage          | `number`                       | 🚫 `false` | 1       | n/a                                      |
-| onPageChange       | `(pageNumber: number) => void` | 🚫 `false` | n/a     | n/a                                      |
+<ArgsTable of={PaginationModelConfigComponent} />
 
-And for the hoisted model pattern, follow this table:
+If you're using the hoisted model pattern, follow this table instead:
 
-| name       | type              | required? | default | recommended                              |
-| ---------- | ----------------- | --------- | ------- | ---------------------------------------- |
-| aria-label | `string`          | ✅ `true` | n/a     | "Pagination" (and translated equivalent) |
-| model      | `PaginationModel` | ✅ `true` | n/a     | n/a                                      |
+<ArgsTable of={PaginationHoistedComponent} />
 
-This component also supports
-[all native HTMLElement props](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes).
```

Comment:
I removed the MDN links throughout the MDX for consistency since we don't use them in other component docs, but I think they're a useful addition. May add them back in a later pass.

Reply:
Yeah, I agree. I think it's a helpful resource, but I'd like it to be consistent too.